amoro-ams/src/main/java/org/apache/amoro/server/table/TableService.java

```diff
@@ -46,4 +47,16 @@ default boolean contains(Long tableId) {
    * @return managed table.
    */
   AmoroTable<?> loadTable(ServerTableIdentifier identifier);
+
+  /** Explore and synchronize table runtimes from catalogs. Intended for periodic sync and tests. */
+  void exploreTableRuntimes();
```

Comment:
Who will call those methods?

Reply:
This method is mentioned in the interface for compatibility with the original test cases in SPI mode, and secondly, this method also needs to be implemented in master-slave mode.